numalogic/monitoring/metrics.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import Optional
+
+from prometheus_client import Counter, Info, Summary
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BaseMetricClass:
```

Comment:
```suggestion
class _BaseMetric:
```
Why `Class` in the name of a class? :)